src/test/java/com/iexec/sms/admin/AdminServiceTests.java

```diff
@@ -149,4 +147,55 @@ void withSQLException(CapturedOutput output) {
         assertTrue(output.getOut().contains("SQL error occurred during restore"));
     }
     // endregion
+
+    // region delete-backup
+    @Test
+    void shouldDeleteBackup(CapturedOutput output) throws IOException {
+        final String backupFileName = "backup.sql";
+        final Path tmpFile = Files.createFile(tempStorageLocation.toPath().resolve(backupFileName));
+        assertAll(
+                () -> assertTrue(adminService.deleteBackupFileFromStorage(tempStorageLocation.getPath(), backupFileName)),
+                () -> assertFalse(tmpFile.toFile().exists()),
+                () -> assertTrue(output.getOut().contains("Successfully deleted backup"))
+        );
+    }
+
+    @Test
+    void shouldFailedDeleteWithBackupFileMissing() throws IOException {
+        final String backupStorageLocation = tempStorageLocation.getCanonicalPath();
+        assertThrows(
+                FileSystemNotFoundException.class,
+                () -> adminService.deleteBackupFileFromStorage(backupStorageLocation, "backup.sql"),
+                "Backup file does not exist"
+        );
+    }
+
+    @Test
+    void shouldFailedToDeleteWithBackupFileOutOfStorage(CapturedOutput output) {
+        assertAll(
+                () -> assertFalse(adminService.deleteBackupFileFromStorage("/backup", "backup.sql")),
+                () -> assertTrue(output.getOut().contains("Backup file is outside of storage file system"))
+        );
+    }
+
+    // endregion
+
+    //region utils
+
+    @Test
+    void testCommonsParametersValidation() {
```

Comment:
```suggestion
    void testCheckCommonParametersValidation() {
```
or
```suggestion
    void testCommonParametersValidation() {
```

Reply:
ok for testCheckCommonParametersValidation